app/models/pipeline/static_analysis.py

```diff
@@ -81,18 +81,9 @@ async def review(self, mr: MergeRequest) -> PipelineResult:
 
```

Comment:
[From AI Code Reviewer][bot] The per-file comment limit of 2 has been removed. Was this intentional? If so, update relevant documentation.